kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -2726,6 +2731,13 @@ private void updateProducerStats(final TopicPartition topicPartition, final int
                 .getCounter(MESSAGE_IN)
                 .add(numMessages);
 
+        if (conversionCount > 0) {
+            requestStats.getStatsLogger()
+                    .scopeLabel(TOPIC_SCOPE, topicPartition.topic())
```

Comment:
Does it miss the `scopeLabel` for `PARTITION_SCOPE`?

Reply:
You are right. I have add it.